endoreg_db/serializers/examination/base.py

```
@@ -0,0 +1,46 @@
+# endoreg_db/serializers.py
+from rest_framework import serializers
+from ...models import Examination, ExaminationType, Finding
```

Ruff (0.12.2)

`...models.Finding` imported but unused

Remove unused import: `...models.Finding`

(F401)